### DISCUSSION TITLE
Triggering Event in iFrame from Parent Window (data-cy)

### DISCUSSION THREAD

**Discussion by ATOPBIO** [cypress-io/cypress#14504] [Questions and Help] [answered]
Hello,

I have a div in an **iFrame:**

div data-v-5a92d3f2="" class="top" **data-cy="toggle-images"**

I am trying to trigger trigger the div from a button in the Parent Window.

Any ideas?

Many thanks in advance

**Comment by bahmutov** [comment, marked as answer]
You would need to access the element as described in https://www.cypress.io/blog/2020/02/12/working-with-iframes-in-cypress/ blog post and call `.trigger(...)`
